Assets/Scripts/Enemies/Enemy.cs

```diff
@@ -14,6 +14,8 @@ public class Enemy : MonoBehaviour
     public bool inKnockback = false;
     public float damagedTime = 0.2f;
 
+    [SerializeField] Animator animator;
+
```

Comment:
Can we add this Awake method to validate the `animator` is present?

```
using UnityEngine.Assertions;
// ...
void Awake() {
  Assert.IsNotNull(animator);
}
```

This **Fail Fast and Hard** pattern will make it easier to notice this bug if we forget to assign the animator to any enemy.